betelgeuse.py

```diff
@@ -209,6 +221,22 @@ def test_case(path, collect_only, project):
                     test_case.update()
 
 
+@cli.command('test-results')
+@click.option(
+    '--path',
+    default='tests',
+    help='Path to the test module or directory.',
+    type=click.Path(exists=True),
+)
+def test_results(path):
+    """Sync test cases with Polarion."""
+    test_summary = parse_test_results(parse_junit(path))
+    summary = " ".join(
```

Comment:
Please prefer single quotes for string to maintain the same style other code uses.